redbot/cogs/mod/kickban.py

```diff
@@ -478,7 +478,7 @@ async def tempban(
                 server_name=guild.name, date=unban_time.strftime("%m-%d-%Y %H:%M:%S")
             )
             if invite:
-                msg += _(" Here is an invite for when your ban expires: {invite_link}").format(
+                msg += _("Here is an invite for when your ban expires: {invite_link}.").format(
```

Comment:
The space here is needed otherwise there will be no space between this string and `"You have been temporarily banned from {server_name} until {date}."`

Also because discord is weird ... I wouldn't add the dot here, or if you do i would add `<>` around the invite link.  because on some clients I've seen discord trying to add following symbols to the URL.

Reply:
So after some #advanced-testing it does seem like invites are super picky. I'll just leave those lines as they are.